_config.yml

```diff
@@ -48,16 +48,16 @@ download:
 
 cdn:
```

Comment:
should we git ignore this file? is it autogenerated?

Reply:
As y'all chatted about in Chapter meeting, this is a part of the Bootstrap build process. Technically this file has important configuration details in it and shouldn't be ignored. The changes to this file are done via The build script to hash the files (some of which we don't need). All that to say... For now I'd just leave it as modified because it doesn't hurt anything.

I'm excited about y'all investigating better improvements for this project. 😄